### PR TITLE
New font: Jaapokki

### PR DESCRIPTION
**Pull request by rstacruz**
http://mikkonuuttila.com/jaapokki/

### CODE REVIEW

File: Casks/font-jaapokki.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'font-jaapokki' do
+  version '1.00'
+  sha256 'aa5f38cbb682df8732d406996e37349b83e47d3c14bae4af2125f2a7784430b7'
+
+  url 'http://mikkonuuttila.com/jaapokki/Jaapokki-font-package-1.0.zip'
```
Comment (adityadalal924) [inline Casks/font-jaapokki.rb]: use `#{version}`and double quotes instead of 1.00

File: Casks/font-jaapokki.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  homepage 'http://mikkonuuttila.com/jaapokki/'
+  license :unknown
+
+  font 'Jaapokki-font-package-1.0/Regular/Jaapokki-Regular.otf'
```
Comment (adityadalal924) [inline Casks/font-jaapokki.rb]: same here

File: Casks/font-jaapokki.rb (hunk 1 above)
Comment (rstacruz) [inline Casks/font-jaapokki.rb]: Version is 1.00 though, not 1.0.

Comment (adityadalal924) [inline Casks/font-jaapokki.rb]: We're a bit flexible with versions, you can just change it to `1.0` instead of `1.00`

**Comment by rstacruz** [comment]
done :)

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8b55bb3b31ee545af0095f8b6c44300cc4651d83. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**